src/views/settings/tabs/RecommendedCapabilitiesTab/context/RecommendedCapabilitiesContext.tsx

```
@@ -0,0 +1,75 @@
+import React, { createContext, FC, ReactNode, useCallback, useContext, useMemo, useState } from 'react';
```

ESLint

[error] Replace `·createContext,·FC,·ReactNode,·useCallback,·useContext,·useMemo,·useState·` with `⏎··createContext,⏎··FC,⏎··ReactNode,⏎··useCallback,⏎··useContext,⏎··useMemo,⏎··useState,⏎`

(prettier/prettier)